controllers/cincinnati_controller.go

```diff
@@ -563,9 +606,8 @@ func (r *ReconcileCincinnati) ensurePolicyEngineRoute(ctx context.Context, reqLo
 		reqLogger.Info("Updating Route", "Namespace", route.Namespace, "Name", route.Name)
 		updated.Spec = route.Spec
 		// We want to allow user to update the TLS cert/key manually on the route and we don't want to override that change.
-		// Keep the existing tls on the route
-		updated.Spec.TLS = tls
-		err = r.client.Update(ctx, updated)
+		// Keep the existing tls on the route                updated.Spec.TLS = tls
```

Comment:
Did you lose a newline here, with the now-dangling, commented `updated.Spec.TLS = tls`?

Reply:
exactly